modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc

```diff
@@ -0,0 +1,749 @@
+
+|MirrorCheckpointConnector
+|A source connector that ensures the mirroring process can resume from where it left off in case of failures. It tracks and emits checkpoints that mirror the offsets of the source and target clusters.
+
+|MirrorHeartbeatConnector
+|A source connector that emits heartbeats to target topics at a regular interval, enabling MirrorMaker to track active topics on the source cluster and to synchronize consumer groups across clusters.
+
+|===
+
+NOTE: If you want to use other connectors, you must create a custom Docker image that includes them as plugins. See <<Install a new connector>>.
+
+== Prerequisites
+
+- A kubernetes cluster. You must have `kubectl` with at least version {supported-kubernetes-version}.
++
+To check if you have `kubectl` installed:
++
+```bash
+kubectl version --short --client
+```
+
+- https://helm.sh/docs/intro/install/[Helm^] installed with at least version {supported-helm-version}.
++
+To check if you have Helm installed:
++
+```bash
+helm version
+```
+
+- For better readability, you need https://stedolan.github.io/jq/download/[jq^] to parse JSON results when using the Kafka Connect REST API.
+
+- An understanding of https://kafka.apache.org/20/documentation.html#connect_overview[Kafka Connect].
+
+== Limitations
+
+- *No TLS or SASL for the Kafka Connect REST API*: All incoming traffic to Kafka Connect, such as from Redpanda Console, is unauthenticated (no SASL) and sent in plain text (no TLS). Although Kafka Connect supports TLS and SASL, the Redpanda Connectors subchart does not. Outgoing traffic from Kafka Connect to Redpanda brokers does support TLS and SASL.
+
+- *Pod replicas*: You cannot configure the number of Pod replicas for Redpanda Connectors. Redpanda Data recommends using an autoscaler such as https://keda.sh/[Keda] to increase the number of Pod replicas automatically when certain conditions, such as high CPU or memory usage, are met.
+
+== Deploy the Helm chart
+
+The Redpanda Helm chart includes Redpanda Connectors as a glossterm:subchart['In Helm, a subchart is a child chart that's packaged within a parent chart.'] so that you can deploy a Redpanda cluster, Redpanda Connectors, and Redpanda Console using a single chart. You can enable and configure the Redpanda Connectors subchart in the `connectors` section of the Helm values.
+
+The subchart includes a Pod that runs Kafka Connect and the built-in connectors. The Pod is managed by a Deployment that you can configure in the Helm values under `connectors.deployment`. Redpanda Console connects to Kafka Connect through the default `redpanda-connectors` Service. Kafka Connect connects to the Redpanda brokers through the default `redpanda` Service.
```

Comment:
Is the uppercase Deployment here indicative of a proprietary aspect of k8s? Otherwise, why would it be capitalized?